src/main/java/eu/bbmri_eric/negotiator/governance/network/Network.java

```diff
@@ -28,7 +28,7 @@
 
 /**
  * Represents a Network entity in the database. A Network has a unique identifier, a URI, a name, a
- * contact email, a set of managers, and a set of resources.
+ * description, a contact email, a set of managers, and a set of resources.
```

Comment:
I would remove the list of attributes from the JavaDoc

Reply:
Fixed in [dd39ca8](https://github.com/BBMRI-ERIC/negotiator/pull/529/commits/dd39ca81c88f841010c840aba8962026ac73eb54)